dev/source/docs/ros-distance-sensors.rst

```diff
@@ -0,0 +1,72 @@
+.. _ros-distance-sensors:
+
+==========================
+ROS distance sensors usage
+==========================
+
+.. warning::
+
+    Distance sensors message are currently support on :
+
+    - Ardupilot Master
+    - Copter >= 3.5.1
```

Comment:
3.4? Downward rangefinder wouldn't send distance sensor message, but it would send for all the other directions. In any case, 3.5 isn't out yet, let alone 3.5.1 :slightly_smiling_face: